Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/Workspaces/PruneContentStreams.feature

```diff
@@ -41,6 +41,7 @@ Feature: If content streams are not in use anymore by the workspace, they can be
     When the command RebaseWorkspace is executed with payload:
       | Key           | Value       |
       | workspaceName | "user-test" |
+      | rebaseErrorHandlingStrategy | "force"               |
```

Comment:
for these tests ... as we dont make changes live and rebase is now a noop if the workspace is not outdated, we made "force" reflect the old behaviour to always fork even if there are no changes in the base.